AutomatedTesting/Gem/PythonTests/Atom/atom_utils/atom_tools_utils.py

```diff
@@ -117,6 +117,22 @@ def select_model_config(asset_path: str) -> None:
     azlmbr.atomtools.EntityPreviewViewportSettingsRequestBus(azlmbr.bus.Broadcast, "LoadModelPresetByAssetId", asset_id)
 
 
+def undo(document_id: azlmbr.math.Uuid) -> bool:
+    return azlmbr.atomtools.AtomToolsDocumentRequestBus(bus.Event, "Undo", document_id)
+
+
+def redo(document_id: azlmbr.math.Uuid) -> bool:
+    return azlmbr.atomtools.AtomToolsDocumentRequestBus(bus.Event, "Redo", document_id)
+
+
```

Comment:
I noticed that the undo and redo functions are used in your new code, but begin_edit and end_edit are not, (unless I'm missing something). Are you going to use them in a future PR?

Reply:
nevermind I just missed them :P